packages/perseus/src/widgets.ts

```diff
@@ -149,22 +175,19 @@ export const getVersionVector = (): {
     [key: string]: Version;
 } => {
     const version: Record<string, any> = {};
-    _.each(_.keys(widgets), function (name) {
+    Object.keys(widgets).forEach((name) => {
         version[name] = getVersion(name);
     });
     return version;
 };
 
-export const getPublicWidgets = (): ReadonlyArray<WidgetExports> => {
-    // TODO(alex): Update underscore.js so that _.pick can take a function.
-    // @ts-expect-error - TS2740 - Type 'Pick<{ [key: string]: Readonly<{ name: string; displayName: string; getWidget?: (() => ComponentType<any>) | undefined; accessible?: boolean | ((props: any) => boolean) | undefined; hidden?: boolean | undefined; ... 10 more ...; widget: ComponentType<...>; }>; }, string>' is missing the following properties from type 'readonly Readonly<{ name: string; displayName: string; getWidget?: (() => ComponentType<any>) | undefined; accessible?: boolean | ((props: any) => boolean) | undefined; hidden?: boolean | undefined; ... 10 more ...; widget: ComponentType<...>; }>[]': length, concat, join, slice, and 18 more.
-    return _.pick(
-        widgets,
-        // @ts-expect-error - TS2345 - Argument of type '(name: string) => boolean | undefined' is not assignable to parameter of type 'Iteratee<string[], boolean, string>'.
-        _.reject(_.keys(widgets), function (name) {
-            return widgets[name].hidden;
-        }),
-    );
+export const getPublicWidgets = (): Record<string, WidgetExports> => {
```

Comment:
The return type was wrong. I wrote the test before changing it and it's always returned an object.